roles/multibench_run/tasks/main.yml

```diff
@@ -0,0 +1,55 @@
+---
+# If this shell cmd ahs a correct rc, it means the multibench_host is ready
+- name: "Check the multi-bench host has the crucible binaries"
+  ansible.builtin.shell: >
+    crucible help
```

Comment:
if the binaries are not installed, it will through an error, but it would be better to check the rc and failed with a proper message when the binaries are missing